rpgJavaInterpreter-core/src/test/resources/smeup/T10_A90.rpgle

```diff
@@ -0,0 +1,57 @@
+     D A90_A2          S             30    INZ
+     D A90_A3          S              8
+     D A90_A4          S              8
+     D A90_A5          S              8
+     D £DBG_Pas        S              3
+     D £DBG_Str        S            100
+     D NNN             S              6  0 INZ(100000)
+     D A90_UP          C                   'ABCDEFGHIJKLMNOPQRS-
+     D                                     TUVWXYZ'
+     D A90_LO          C                   'abcdefghijklmnopqrs-
+     D                                     tuvwxyz'
+      *
+     C*                   EVAL      £DBG_Pas='P01'
```

Comment:
Why is this step commented?

Reply:
I created from mulangt90 but I noticed afterward that there was a case not managed. So I commented the code that didn't work. It should be fixed in the next commit.